R/markdown.R

```diff
@@ -3,9 +3,9 @@ markdown <- function(path = NULL, ..., strip_header = FALSE) {
   on.exit(file_delete(tmp), add = TRUE)
 
   if (rmarkdown::pandoc_available("2.0")) {
-    from <- "markdown_github-hard_line_breaks+smart+auto_identifiers+tex_math_dollars+tex_math_single_backslash+markdown_in_html_blocks"
+    from <- "markdown_github-hard_line_breaks+smart+auto_identifiers+tex_math_dollars+tex_math_single_backslash+markdown_in_html_blocks+header_attributes"
   } else if (rmarkdown::pandoc_available("1.12.3")) {
-    from <- "markdown_github-hard_line_breaks+tex_math_dollars+tex_math_single_backslash"
+    from <- "markdown_github-hard_line_breaks+tex_math_dollars+tex_math_single_backslash+header_attributes"
```

Comment:
Are you sure this is supported in 1.12? I think you'd be better off just leaving this block alone.

Reply:
From the release notes and looking at the past docs, [header attributes were introduced in pandoc 1.10](https://pandoc.org/releases.html#pandoc-1.10-2013-01-19) but I've taken it out to be safe regardless.